packages/core-utils/src/list-utils.ts

```diff
@@ -0,0 +1,5 @@
+export const append = <Type>(value: Type): ((list: Type[]) => Type[]) => {
```

Comment:
lets add a jsdoc method comment to document intended usage with:
- note that it is "curried" and returns a function expecting array to append item to
- note that it honors immutable pattern (new array returned versus modifying input array)
- example call usage.